x-pack/legacy/plugins/monitoring/public/components/no_data/explanations/collection_enabled/collection_enabled.js

```diff
@@ -39,10 +39,10 @@ export class ExplainCollectionEnabled extends React.Component {
     const { enabler } = this.props;
     enabler.enableCollectionEnabled();
 
-    // wait 19 seconds, show link to reload
+    // wait 22 seconds, show link to reload
     this.waitedTooLongTimer = setTimeout(() => {
       this.setState({ waitedTooLong: true });
-    }, 19 * 1000);
+    }, 22 * 1000);
```

Comment:
What was the reasoning around changing this?

Reply:
If you enable monitoring right after the check, it can show the "still waiting" state for a brief second. Figured we'd give it more wiggle room to only show that when it is legitimately taking a long time / timing-out.

I don't mind changing it back though

Reply:
Maybe we can address this in a follow up PR, but we should possibly look into a way to remove these magic numbers (I realize you didn't add them originally)